docs/specs/resolve.yml

```diff
@@ -49,6 +49,26 @@ outputs:
   docs/b.json: |
     { "content": "<p>Link to <a href=\"#\">self</a></p>" }
 ---
+# Links to self bookmarks in the included file
+inputs:
+  docfx.yml: |
+    files: docs/a.md
+  docs/a.md: |
+    # heading
+    [!INCLUDE[](includes/b.md)]
+  docs/includes/b.md: |
+    ## test
+    Link to [valiad self](#test)
+    Link to [valiad parent](../a.md/#heading)
+    Link to [invalid self](#heading1)
+    Link to [invalid parent](../a.md/#heading2)
+outputs:
+  docs/a.json: |
+    { "content": "<h2 id=\"test\">test</h2>\n<p>Link to <a href=\"#test\">valiad self</a>\nLink to <a href=\"a#heading\">valiad parent</a>\nLink to <a href=\"#heading1\">invalid self</a>\nLink to <a href=\"a#heading2\">invalid parent</a></p>" }
+  build.log: |
+    ["suggestion","internal-bookmark-not-found","Cannot find bookmark '#heading2' in 'docs/includes/b.md', did you mean '#heading'?","docs/includes/b.md","",5,9]
```

Comment:
it's the #heading2 is treated as internal bookmark, then could we add a case `Link to [valiad parent](#heading)`

Reply:
we don't recommend the user to use selfbookmark to reference the  parent

---
In reply to: [269406727](https://github.com/dotnet/docfx/pull/4247#discussion_r269406727) [](ancestors = 269406727)